### PR TITLE
mlucas-DOCU-1480

### PR DESCRIPTION
**Pull request by lucasdigital**
Fixes/improvements for the auth instructions.

Changes previewed here:
https://github.com/WANdisco/wandisco-documentation-ldm/blob/mlucas-DOCU-1480/docs/configuration-hvm.md#security

https://github.com/WANdisco/wandisco-documentation-ldm/blob/mlucas-DOCU-1480/docs/configuration-ui.md#basic-authentication-compatibility

### CODE REVIEW

File: docs/configuration-ui.md [hunk 1]
```diff
@@ -154,19 +154,29 @@ Follow the steps below to configure the LiveData UI to work with LiveData Migrat
 1. On the LiveData UI host, run the following command:
 
    ```text
-   livedata-ui encryptor
+   # livedata-ui encryptor
+
+   14:30:15.013 [main] DEBUG com.wandisco.oneui.config.properties.util.AdditionalPropertiesFileUtil
+   - Spring config additional location is set to: /etc/wandisco/ui/application-prod.properties
+   1. Encrypt a string
```
Comment (AshVic) [inline docs/configuration-ui.md:161]: Add periods at the end of the list items.

File: docs/configuration-ui.md [hunk 2]
```diff
@@ -154,19 +154,29 @@ Follow the steps below to configure the LiveData UI to work with LiveData Migrat
...
+   2. Encrypt and save a property to /etc/wandisco/ui/application-prod.properties
+   3. Setup LDM Basic Auth
```
Comment (AshVic) [inline docs/configuration-ui.md:163]: Set up is two words if a verb.

Write LDM as liveData Migrator.

basic auth.

File: docs/configuration-ui.md [hunk 3]
```diff
@@ -154,19 +154,29 @@ Follow the steps below to configure the LiveData UI to work with LiveData Migrat
...
+   4. Setup HVM Basic Auth
```
Comment (AshVic) [inline docs/configuration-ui.md:164]: Hive Migrator basic auth.

File: docs/configuration-ui.md [hunk 4]
```diff
@@ -154,19 +154,29 @@ Follow the steps below to configure the LiveData UI to work with LiveData Migrat
...
+   5. Setup Livedata UI LDAP authentication
```
Comment (AshVic) [inline docs/configuration-ui.md:165]: Live**D**ata UI

File: docs/configuration-ui.md [hunk 5]
```diff
@@ -154,19 +154,29 @@ Follow the steps below to configure the LiveData UI to work with LiveData Migrat
...
+   6. Exit
+   Please select an option:
```
Comment (AshVic) [inline docs/configuration-ui.md:167]: We don't use "please" generally.

File: docs/configuration-ui.md [hunk 6]
```diff
@@ -154,19 +154,29 @@ Follow the steps below to configure the LiveData UI to work with LiveData Migrat
...
    ```
 
-1. Select the `Setup LDM Basic Auth` option when the menu appears.
+1. Select option `3. Setup LDM Basic Auth`.
```
Comment (AshVic) [inline docs/configuration-ui.md:170]: Set up LiveData Migrator basic auth.

File: docs/configuration-ui.md [hunk 7]
```diff
@@ -176,20 +186,29 @@ Follow the steps below to configure the LiveData UI to work with HiveMigrator ba
 1. On the LiveData UI host, run the following command:
 
    ```text
-   livedata-ui encryptor
+   # livedata-ui encryptor
+
+   14:30:15.013 [main] DEBUG com.wandisco.oneui.config.properties.util.AdditionalPropertiesFileUtil
+   - Spring config additional location is set to: /etc/wandisco/ui/application-prod.properties
+   1. Encrypt a string
```
Comment (AshVic) [inline docs/configuration-ui.md:193]: See comments above for adding periods to list items and other edits.

File: docs/configuration-ui.md [hunk 8]
```diff
@@ -176,20 +186,29 @@ Follow the steps below to configure the LiveData UI to work with HiveMigrator ba
...
+   2. Encrypt and save a property to /etc/wandisco/ui/application-prod.properties
+   3. Setup LDM Basic Auth
+   4. Setup HVM Basic Auth
+   5. Setup LiveData UI LDAP authentication
+   6. Exit
+   Please select an option:
    ```
-
-1. Select the `Setup HVM Basic Auth` option when the menu appears.
+1. Select option `4. Setup HVM Basic Auth`.
```
Comment (AshVic) [inline docs/configuration-ui.md:201]: See above comments.

**Review by AshVic — CHANGES_REQUESTED** [review]

File: docs/configuration-ui.md (hunk 1 above)
Comment (lucasdigital) [reply docs/configuration-ui.md:161]: This is a direct copy of the CLI. If I fix it then it won't match. Do I still fix it and live with the disconnect?

Screenshot 2021-11-09 at 15 51 12
.

**Comment by lucasdigital** [comment]
I've agreed with Ashleigh that we'll remove the offending text to get these changes merged. I'll comment on INC-4209 that a doc task should be raised when the CLI tool's wording is updated.

**Pull request merged**